components/typography/typography.html

```diff
@@ -5,8 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Typography</title>
-    <link rel="stylesheet" href="./typography.css" />
-    <link rel="stylesheet" href="../utility.css" />
+    <link rel="stylesheet" href="typography.css" />
+    <link rel="stylesheet" href="/utility.css" />
```

Comment:
If the file is in same directory then "./file.html" or "file.html". Either use single dot and single slash or don't use them both(if the file is in same directory). Don't use only slash. Just a suggestion.